commcare_export/commcare_hq_client.py

```diff
@@ -83,7 +83,19 @@ def authenticated(self, username=None, password=None, mode=AUTH_MODE_SESSION):
         else:
             raise Exception('Unknown auth mode: %s' % mode)
 
-        return CommCareHqClient(url=self.url, project=self.project, version=self.version, session=session, auth=auth)
+        return self._clone_with(session=session, auth=auth)
+
+    def _clone_with(self, **kwargs):
+        kwargs.setdefault('version', self.version)
+        kwargs.setdefault('session', self.__session)
+        kwargs.setdefault('auth', self.__auth)
```

Comment:
It looks like `_clone_with` is called in one place, and `session` and `auth` are both provided.  Do you expect this to be called elsewhere where you may want the default values?

Reply:
No I guess not: 59df155